src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -287,23 +291,39 @@ private static User makeUser(User user, ExtImpRubicon rubiconImpExt) {
                 ? RubiconUserExtRp.of(visitor)
                 : null;
 
-        final ExtUserDigiTrust userExtDt = user != null && user.getExt() != null
-                ? getExtUserDigiTrustFromUserExt(user.getExt())
-                : null;
+        final ExtUser extUser = user != null ? getExtUser(user.getExt()) : null;
+
+        final ExtUserDigiTrust userExtDt = extUser != null ? extUser.getDigitrust() : null;
+
+        final String consent = extUser != null ? extUser.getConsent() : null;
 
-        if (userExtRp != null || userExtDt != null) {
+        if (userExtRp != null || userExtDt != null || consent != null) {
             result = user.toBuilder()
-                    .ext(Json.mapper.valueToTree(RubiconUserExt.of(userExtRp, userExtDt)))
+                    .ext(Json.mapper.valueToTree(RubiconUserExt.of(userExtRp, consent, userExtDt)))
                     .build();
         }
 
         return result;
     }
 
-    private static ExtUserDigiTrust getExtUserDigiTrustFromUserExt(ObjectNode extNode) {
+    private static Regs makeRegs(Regs regs) {
+        final ExtRegs extRegs = regs != null ? getExtRegs(regs.getExt()) : null;
+        final Integer gdpr = extRegs != null ? extRegs.getGdpr() : null;
+        return gdpr != null ? Regs.of(null, Json.mapper.valueToTree(RubiconRegsExt.of(gdpr))) : null;
+    }
+
+    private static ExtRegs getExtRegs(ObjectNode extNode) {
+        try {
+            return extNode != null ? Json.mapper.treeToValue(extNode, ExtRegs.class) : null;
+        } catch (JsonProcessingException e) {
+            logger.warn("Error occurred while parsing bidrequest.regs.ext", e);
+            throw new PreBidException(e.getMessage());
```

Comment:
Could you please pass an exception as well  _`throw new PreBidException(e.getMessage(), e);`_ and on the line 328 that was written by me.